setup.py

```diff
@@ -52,51 +65,52 @@ def get_commitish():
 setup(
     name="pdp",
     description="PCIC's Data Portal (pdp): the server software to run the "
-                "entire web application",
+    "entire web application",
     keywords="opendap dods dap open data science climate meteorology "
-             "downscaling modelling",
-    packages=['pdp', 'pdp.portals'],
+    "downscaling modelling",
+    packages=["pdp", "pdp.portals"],
     version="%s+%s" % (__version__, __revision__),
     url="http://www.pacificclimate.org/",
     author="James Hiebert",
     author_email="hiebert@uvic.ca",
-    install_requires=['flask',
-                      'beaker',
-                      'genshi',
-                      'static',
-                      'pdp-util >=0.2.2',
-                      'pydap.handlers.hdf5 >=0.5',
-                      'pydap.responses.netcdf >=0.5',
-                      'pydap.responses.xls',
-                      'pydap.responses.aaigrid >=0.5',
-                      'pydap.handlers.sql',
-                      'ga-wsgi-client',
-                      'slimit',
-                      'netCDF4<1.5.6'],
-    tests_require=['webob',
-                   'pytest',
-                   'xlrd',
-                   'pillow',
-                   'numpy',
-                   'BeautifulSoup4'],
-    scripts=['scripts/rast_serve.py'],
-    package_dir={'pdp': 'pdp'},
-    package_data={'pdp': ['templates/*.html',
-                          'resources/hydro_stn_archive.yaml',
-                          'resources/hydro_stn_cmip5.yaml'] +
-                  recursive_list('pdp/', 'pdp/static') +
-                  recursive_list('pdp/', 'pdp/docs/html')},
-    cmdclass={
-        'test': PyTest,
-        'build_sphinx': BuildDoc
-    },
-    command_options={
-        'build_sphinx': {
-            'build_dir': ('setup.py', 'pdp/docs')
-        }
+    install_requires=[
+        "flask",
+        "beaker",
+        "genshi",
+        "static",
+        "pdp-util",
```

Comment:
Should `pdp-util` have a hard requirement of `>=` some version?

Reply:
That's perhaps a bigger discussion than fits here. I've taken the discussion to our Zulip.